src/cli/options.rs

```diff
@@ -522,7 +526,7 @@ impl Options {
     }
 
     fn make_cas_parents_request_sc<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
-        let mut sub = SubCommand::with_name("request").about("Show RFC8183 Child Request XML");
+        let mut sub = SubCommand::with_name("request").about("Show idexchange:: Child Request XML");
```

Comment:
You probably didn't mean to change this ;-)

Reply:
nope, fixed